esmvaltool/recipes/recipe_quantilebias.yml

```diff
@@ -0,0 +1,42 @@
+# recipe_quantilebias.yml
+---
+
+documentation:
+  description: |
+    Tool for calculation of precipitation quantile bias
+
+  authors:
+    - arno_en
+    - hard_jo
+
+  references:
+    - mehran14jgr
+
+  projects:
+    - c3s-magic
+
+datasets:
+  - {dataset: EC-EARTH,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1997, end_year: 1997}
+
+preprocessors:
+  preproc:
+    extract_levels: false
+
+diagnostics:
+  quantilebias:
+    description: tool for calculation of precipitation quantile bias 
+    variables:
+      pr:
+        preprocessor: preproc
+        reference_dataset: "EC-EARTH"
+        mip: day
+        field: T2Ds
+    scripts:
+      main:
+        script: quantilebias/quantilebias.R
+        perc_lev: 75
+        # reference global precipitation monthly climatology data:
+        # suggested dataset from the GPCP project at https://precip.gsfc.nasa.gov/
+        # already converted dataset can be downloaded at http://wilma.to.isac.cnr.it/arnone/data/gpcp_22.nc 
```

Comment:
Would it be possible to publish this converted dataset on Zenodo, as suggested in issue #670? If not, could you provide a tool for doing the required conversion?

Reply:
I had missed that issue, good point. I need to check with their license. Also, it is an observational dataset and I saw @mattiarighi commenting not to add obs?

Reply:
The comment of @mattiarighi was about observational datasets where the license does not allow re-publishing I think.

Publishing the data on Zenodo should not be different from publishing it on your personal website from a license point of view. I think it should be possible for users to run your recipe without buying an IDL license, so indeed an IDL routine is not so useful.

Reply:
GPCP data should be freely available and already cmorized via the obs4mips project, so there should be no need to publish them elsewhere.

Reply:
@mattiarighi yes, correct, I found it in obs4mips. So how do we deal with it? Can I load it as a reference dataset from the recipe also in v2.0? Is there an example somewhere I can follow?

Reply:
Sure, see:
https://github.com/ESMValGroup/ESMValTool/blob/847b44079804dd525d9a1f80bf2e830ab5d69f2f/esmvaltool/recipes/recipe_perfmetrics_CMIP5.yml#L1024

Reply:
@mattiarighi @bouweandela any quick suggestion on why esmvaltool is crashing when attempting to find/read the GPCP file?

Data file path and name:
`/work/users/arnone/data/Tier1/GPCP-SG/pr_GPCP-SG_L3_v2.3_197901-201710.nc`

Dataset in recipe:
`  - {dataset: GPCP-SG, project: obs4mips, level: L3, version: v2.3, tier: 1}`

OBS directory in config-user.yml:
` OBS: /work/users/arnone/data`

Instructions in config-developer.yml:
`obs4mips:
  input_dir:
    default: 'Tier[tier]/[dataset]'
  input_file: '[short_name]_[dataset]_[level]_[version]_*.nc'
`

Reply:
See #757 on preproc failing with GPCP and possibly similar dataset from obs4mips.

Reply:
Can you implement this correction in a fix file?
Just create a directory `obs4mips` [here](https://github.com/ESMValGroup/ESMValTool/tree/version2_development/esmvaltool/cmor/_fixes) and add a `GPCP-SG.py` with the fix routine.

Reply:
I implemented a nice working fix file (it took me a while to understand that it should be named `GPCP_SG.py` and not `GPCP-SG.py`)
Would you prefer me to put into this branch or to create a separate branch on purpose with its own PR ?

Reply:
Meanwhile @jvegasbsc implemented a more general fix to address this kind of problems (#757).
If that works for your case we do not need a fix file.

Reply:
Yes, #757 works for our case. Well, at least I learnt how to write a fix ;)